openapi3filter/validate_request.go

```diff
@@ -37,6 +39,8 @@ func ValidateRequest(c context.Context, input *RequestValidationInput) error {
 	operationParameters := operation.Parameters
 	pathItemParameters := route.PathItem.Parameters
 
+	var me openapi3.MultiError
```

Comment:
Shaving a bit here but if you don't mind: group this declaration with the `var err error` above please.